### PR TITLE
PT-BR Translation for javascript-conventions/using-ECMA5-in-nodejs

### PR DESCRIPTION
**Pull request by romulo94**
@nodejs/nodejs-pt：Please help me to have a review, thanks!

### CODE REVIEW

**Comment by romulo94** [comment]
@khaosdoctor
@glaucia86

File: locale/pt-br/knowledge/javascript-conventions/using-ECMA5-in-nodejs.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  - core
+  - builtin
+  - globals
+difficulty: 2
+layout: knowledge-post.hbs
+---
+
+<!-- When developing in the browser there are many wonderful built in JavaScript functions that we can't use because certain browsers don't implement them.  As a result, most developers never use them.  In Node, however we can assume that everyone has the same JavaScript implementation and as such can use these wonderful functions and not implement them over and over in our own libraries. -->
+
+Ao desenvolver no navegador, há muitas funções maravilhosas construídas em JavaScript que não podemos usar, porque certos navegadores não as implementam. Como resultado, a maioria dos desenvolvedores nunca os usa. No Node, no entanto, podemos assumir que todos têm a mesma implementação de JavaScript e, como tal, podem usar essas funções maravilhosas e não implementá-las repetidamente em nossas próprias bibliotecas.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/javascript-conventions/using-ECMA5-in-nodejs.md]:
```suggestion
Ao desenvolver no navegador, há muitas funções maravilhosas construídas em JavaScript que não podemos usar, porque certos navegadores não as implementam. Como resultado, a maioria dos desenvolvedores nunca as usa. No Node, no entanto, podemos assumir que todos têm a mesma implementação de JavaScript e, como tal, podem usar essas funções maravilhosas e não implementá-las repetidamente em nossas próprias bibliotecas.
```

File: locale/pt-br/knowledge/javascript-conventions/using-ECMA5-in-nodejs.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+<!-- The following is a list of some interesting api bits that aren't considered safe to use in a web setting but are built in to node's V8 engine. -->
+
+A seguir está uma lista de alguns pedaços interessantes da API que não são considerados seguros para uso em uma configuração da Web, mas que são incorporados ao mecanismo V8 do node.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/javascript-conventions/using-ECMA5-in-nodejs.md]:
```suggestion
A seguir está uma lista de algumas partes interessantes da API que não são considerados seguros para uso no desenvolvimento para a Web, mas que são nativos ao mecanismo V8 do node.
```

File: locale/pt-br/knowledge/javascript-conventions/using-ECMA5-in-nodejs.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+  <!-- - `Object.getOwnPropertyDescriptor(obj, property)` - Returns an object with keys describing the description of a property (value, writable, enumerable, configurable) -->
+- `Object.getOwnPropertyDescriptor(obj, property)` - Retorna um objeto com chaves descrevendo a descrição de uma propriedade (valor, gravável, enumerável, configurável).
+  <!-- - `Object.preventExtensions(obj)` - Prevents any new properties from being added to the given object. -->
+- `Object.preventExtensions(obj)` -Impede que quaisquer novas propriedades sejam adicionadas ao objeto fornecido.
+  <!-- - `Object.isExtensible(obj)` - Checks if Object.preventExtensions() has been called on this object. -->
+- `Object.isExtensible(obj)` - Verifica se Object.preventExtensions() foi chamado neste objeto.
+  <!-- - `Object.seal(obj)` - Prevents code from adding or deleting properties, or changing the descriptors of any property on an object. Property values can be changed however. -->
+- `Object.seal(obj)` - Impede que o código adicione ou exclua propriedades ou altere os descritores de qualquer propriedade em um objeto. Valores de propriedade podem ser alterados no entanto.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/javascript-conventions/using-ECMA5-in-nodejs.md]:
```suggestion
- `Object.seal(obj)` - Impede que o código adicione ou exclua propriedades ou altere os descritores de qualquer propriedade em um objeto. No entanto, valores de propriedade podem ser alterados.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]

File: locale/pt-br/knowledge/javascript-conventions/using-ECMA5-in-nodejs.md (hunk 2 above)
Comment (romulo94) [reply locale/pt-br/knowledge/javascript-conventions/using-ECMA5-in-nodejs.md]: @khaosdoctor it's done

**Review by khaosdoctor — APPROVED** [review]
LGTM 👍

**Comment by khaosdoctor** [comment]
cc: @MaledongGit @Trott

**Comment by ghost** [comment]
Please also see #2278. We don't need to do translations for knowledge yet, if you have other feedbacks, please leave them here :)

**Pull request closed without merging**